src/main/java/htsjdk/samtools/cram/ref/ReferenceSource.java

```diff
@@ -63,7 +62,7 @@ private ReferenceSource() {
     }
 
     public ReferenceSource(final File file) {
-        this(file == null ? null : file.toPath());
+        this(IOUtil.toPath(file));
     }
 
     public ReferenceSource(final Path path) {
```

Comment:
With a minor change to this constructor, you can make the field `rsFile` `final`, which would match its current usage in the code